src/productcatalogservice/product_catalog.go

```diff
@@ -27,6 +28,7 @@ import (
 
 type productCatalog struct {
 	catalog pb.ListProductsResponse
+	pb.UnimplementedProductCatalogServiceServer
```

Comment:
Context for adding this `pb.UnimplementedProductCatalogServiceServer` line: https://github.com/grpc/grpc-go/issues/3794#issuecomment-720599532.
**tl;dr**: It's a gRPC best practice, while a [lot of developers don't necessarily agree with](https://github.com/grpc/grpc-go/issues/3669).